eng/pipelines/build.yml

```diff
@@ -29,20 +30,17 @@ jobs:
           DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
           DOTNET_CLI_TELEMETRY_OPTOUT: 1
           DOTNET_MULTILEVEL_LOOKUP: 0
-      - pwsh: ./eng/CodeChecks.ps1
-        displayName: "Check if code is up-to-date"
-      - script: |
-          dotnet test AutoRest.CSharp.V3.sln
-        displayName: "Test"
-        env:
-          DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
-          DOTNET_CLI_TELEMETRY_OPTOUT: 1
-          DOTNET_MULTILEVEL_LOOKUP: 0
-      - script: |
-          dotnet pack /t:NpmPackage src/AutoRest.CSharp.V3/AutoRest.CSharp.V3.csproj -o $(Build.ArtifactStagingDirectory)
-        displayName: "Publish NPM package"
-      - task: PublishPipelineArtifact@1
-        condition: succeeded()
-        inputs:
-          artifactName: packages
-          path: $(Build.ArtifactStagingDirectory)
\ No newline at end of file
+#      - pwsh: ./eng/CodeChecks.ps1
+#        displayName: "Check if code is up-to-date"
+#      - script: |
+#          dotnet test AutoRest.CSharp.V3.sln
+#        displayName: "Test"
+#        env:
+#          DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+#          DOTNET_CLI_TELEMETRY_OPTOUT: 1
+#          DOTNET_MULTILEVEL_LOOKUP: 0
```

Comment:
Why are code checks and testing commented out?

Reply:
Testing artifact. Undoing.